data-prepper-plugins/grok-processor/src/main/resources/grok-patterns/patterns

```diff
@@ -0,0 +1,15 @@
+ALB_URI: (%{WORD:protocol})?(://)?(%{IPORHOST:domain})?(:)?(%{INT:http_port})?(%{GREEDYDATA:request_uri})?
+ALB_REQUEST_LINE (%{NOTSPACE:http_method})? (%{NOTSPACE:http_uri})? (%{NOTSPACE:http_version})?
```

Comment:
TIOLI: I don't like the `ALB_REQUEST_LINE` as key here, it's not clear enough. Maybe something like `ALB_HTTP_REQUEST`.  Same for other entries with `REQUEST_LINE`